src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -373,3 +375,59 @@ def test_create_storage_account_with_files_aadds_true(self, resource_group):
 
         self.assertIn('azureFilesIdentityBasedAuthentication', result)
         self.assertEqual(result['azureFilesIdentityBasedAuthentication']['directoryServiceOptions'], 'AADDS')
+
+
+class RoleScenarioTest(LiveScenarioTest):
```

Comment:
why do we need this class?

Reply:
Test if users are beyond a ServicePrincipal's capacity.